docs/reference/api/connectivity/lorawan/loraradio.md

```diff
@@ -0,0 +1,12 @@
+<h2 id="loraradio-api">LoRaRadio</h2>
+
+`LoRaRadio` is a pure virtual class that defines APIs for a LoRa radio driver.
+
+An implementation of this class is passed to the Arm Mbed LoRaWAN stack. Please see existing [Mbed OS supported drivers](https://github.com/ARMmbed/mbed-semtech-lora-rf-drivers) for more information.
+
+### LoRaRadio class reference
+
+[![View code](https://www.mbed.com/embed/?type=library)](http://os-doc-builder.test.mbed.com/docs/v5.8/mbed-os-api-doxy/class_l_o_r_a_radio.html)
```

Comment:
I think it will work after the release. I am not sure. I just followed what others had been doing here. Seemingly, the script to fetch doxygen from Mbed-OS would make it work during the release process.

Reply:
This is valid link currently:
https://os-doc-builder.test.mbed.com/docs/development/mbed-os-api-doxy/class_lo_ra_radio.html